articles/storage/storage-managed-disks-overview.md

```diff
@@ -144,7 +144,7 @@ For more information about Managed Disks, please refer to the following articles
 
 * [Attach a managed data disk to a Windows VM using PowerShell](../virtual-machines/windows/attach-disk-ps.md)
 
-* [Add a managed disk to a Linux VM](../virtual-machines/linux/quick-create-cli.md)
+* [Add a managed disk to a Linux VM](../virtual-machines/linux/add-disk#attach-a-managed-disk)
```

Comment:
Can you please link this to just ../virtual-machines/linus/add-disk  rather than a section in that document?